drivers/espi/host_subs_npcx.c

```diff
@@ -1055,6 +1056,17 @@ void npcx_host_init_subs_host_domain(void)
 	/* Enable SHM direct memory access */
 	host_c2h_write_io_cfg_reg(EC_CFG_IDX_CTRL, 0x01);
 	}
+
+	if (IS_ENABLED(CONFIG_ESPI_PERIPHERAL_UART)) {
+		/* Select Serial Port banks which LDN are 0x03. */
+		host_c2h_write_io_cfg_reg(EC_CFG_IDX_LDN, EC_CFG_LDN_SP);
+		/* Enable SIO_CLK */
+		host_c2h_write_io_cfg_reg(0x29, host_c2h_read_io_cfg_reg(0x29) | BIT(2));
+		/* Enable Bank Select */
+		host_c2h_write_io_cfg_reg(0xf0, host_c2h_read_io_cfg_reg(0xf0) | BIT(7));
```

Comment:
should there be a define for the magics here?

Reply:
Done.